### DISCUSSION TITLE
It is possible to add directory that serve static files?

### DISCUSSION THREAD

**Discussion by n2ptune** [vercel/next.js#24225] [Help] [answered]
I know that placing the file in the /public directory creates a static file in the format /abc.txt. But what directory can I turn into a static serving directory?

I also want to serve static assets in the /public directory and the /src/assets/images directory.

Is this possible?

**Comment by akellbl4** [comment, marked as answer]
Hi,
As far as I know, you are not able to serve specific folders with Next.js. It serves only `public` directory and you have to place everything there. But you can copy files before build there.

**Reply by n2ptune** [reply]
Hi. @akellbl4 
Thank you for replying.
If what I understand is correct, can I run `mv` or `cp` before `next build`?

**Reply by akellbl4** [reply]
Yep, right ;) Also, I could suggest adding files that should be copied in git ignore `public/...`. It'll prevent duplicates of the files and you won't commit it accidentally to the repo.

**Reply by n2ptune** [reply]
@akellbl4 Thank you very so much it was many helpful for me!

**Reply by akellbl4** [reply]
@n2ptune you are welcome! I'm always happy to help ;)
